core/jobs.py

```diff
@@ -844,6 +850,9 @@ def enqueue(cls, job_id, additional_job_params=None):
 
 
 class MultipleDatastoreEntitiesInputReader(input_readers.InputReader):
+    """Multiple Datastore Entities Input Reader, these are used to read values
```

Comment:
No need to rewrite the name of the class in the docstring. Something like this suffices:

This class is used to read values from multiple classes in the datastore and pass them to mapper functions in MapReduce jobs.

(Note the mention of multiple classes -- this is why this class is written in the first place.)

Reply:
done.